docs/contributor/arch/002-fluent-bit-without-file-system-buffer.md

```diff
@@ -0,0 +1,63 @@
+# 1. Fluent Bit without File-System Buffer
+
+Date: 2023-11-23
+
+## Status
+
+Proposed
+
+## Context
+
+The Fluent Bit configuration that we currently generate with the telemetry-manager from a LogPipeline uses a `tail`
```

Comment:
let's not split sentences across lines 

```suggestion
The Fluent Bit configuration that we currently generate with the telemetry-manager from a LogPipeline uses a `tail` input plugin to read container logs, splits the log stream into multiple pipelines using a `rewrite_tag` filter, applies additional pipeline specific filters, and sends the log stream to the configured output. This setup decouples the input and outputs so that a blocked output does not affect any other outputs. Consequently, reading logs by the tail plugin is never paused. The `rewrite_tag` filter uses a persistent file-system buffer to prevent log loss. The persistent buffer is limited to 1 GB to ensure that Kubernetes nodes do not run out of disk space.
```